src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/RoleCaseload.kt

```diff
@@ -0,0 +1,60 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa
+
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Type
+import java.io.Serializable
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Embeddable
+import javax.persistence.EmbeddedId
+import javax.persistence.Entity
+import javax.persistence.FetchType.LAZY
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Suppress("DataClassEqualsAndHashCodeInspection")
+@Embeddable
+data class RoleCaseloadPk(
+  @Column(name = "ROLE_ID", nullable = false)
+  var roleId: Long,
+  @Column(name = "LOCAL_AUTHORITY_CODE", nullable = false)
+  var caseload: String,
+) : Serializable
+
+@Entity
+@Table(name = "LAA_GRANTED_ROLES")
```

Comment:
Curious, does this table map what roles can be assigned to a caseload? So can Brixton have a different set roles that can be assigned by LSAs compared to say Wandsworth?